model/Licensing/Classes/DisjunctiveLicenseSet.md

```diff
@@ -0,0 +1,31 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DisjunctiveLicenseSet
+
+## Summary
+
+Portion of a LicenseExpression representing a set of licensing information
+where only any one of the elements applies.
+
+## Description
+
+A DisjunctiveLicenseSet indicates that _only one_ of its subsidiary
+LicenseExpressions is required to apply. In other words, a
+DisjunctiveLicenseSet of two or more licenses represents a licensing
+situation where _only one_ of the specified licenses are to be complied with.
+A consumer of SPDX data would typically understand this to permit the recipient
+of the licensed content to choose which of the corresponding license they
+would prefer to use. It is represented in the SPDX License Expression Syntax
+by the `OR` operator.
+
+## Metadata
+
+- name: DisjunctiveLicenseSet
+- SubclassOf: LicenseExpression
```

Comment:
Similar comment to above on difference in naming from 2.X - Note that since `AnyLicenseInfo` is abstract, the impact of changing the name is not that great, but it would be nice to be consistent.